File: 1689-partitioning-into-minimum-number-of-deci-binary-numbers/1689-partitioning-into-minimum-number-of-deci-binary-numbers.cpp
class Solution {
public:
    int minPartitions(string n) {
        int ret = 0;
        for(int x = 0; x<n.size(); x++){
            int digit = n[x]-'0';
            ret = max(ret, digit);
        }
        return ret;
        
    }
};

File: 1689-partitioning-into-minimum-number-of-deci-binary-numbers/1689-partitioning-into-minimum-number-of-deci-binary-numbers.js
/**
 * @param {string} n
 * @return {number}
 */
var minPartitions = function(n) {
    let mx = '0';
    for(let x = 0; x<n.length; x++){
        mx = Math.max(n[x] , mx);
    }
    return mx-'0';
};